src/components/Explore.tsx

```diff
@@ -0,0 +1,263 @@
+  const colourCodeGenreLabels = [
+    "bg-red-600/20",
+    "bg-orange-500/20",
+    "bg-yellow-500/20",
+    "bg-lime-500/20",
+    "bg-cyan-500/20",
+    "bg-purple-400/20",
+    "bg-green-700/20",
+    "bg-pink-500/20",
+  ];
+
+  // Icons assoicated with each genre.
+  const comicGenreIcons = [
+    <PiLightningFill className="text-red-600 text-lg" />,
+    <PiSwordFill className="text-orange-500 text-lg" />,
+    <FaLaugh className="text-yellow-500 text-lg" />,
+    <FaHourglassHalf className="text-lime-500 text-lg" />,
+    <FaWandMagicSparkles className="text-cyan-500 text-lg" />,
+    <GiMagnifyingGlass className="text-purple-400 text-lg" />,
+    <PiAlienFill className="text-green-700 text-lg" />,
+    <IoSunnySharp className="text-pink-500 text-lg" />,
+  ];
+
+  // Controls what happens when one of the genres are selected.
+  function handleGenreClick(genre: string) {
+    setSelectedGenre(genre.toLowerCase());
+  }
+
+  function handleSearch(comicName: string) {
+    setSearchedComicName(comicName);
+  }
+
+  return (
+    <>
+      <Formik
+        initialValues={{
+          searchString: "",
+        }}
+        validationSchema={comicSearchSchema}
+        onSubmit={(values, { setSubmitting }) => {
+          handleSearch(values.searchString);
+          setSubmitting(false);
+        }}
+      >
+        {({ errors, touched }) => (
+          <Form className="w-full flex flex-row flex-wrap justify-between mb-8">
+            <Field
+              className="w-4/5 pl-4 py-2 rounded-2xl h-8 outline-none border-none bg-purple-grey"
+              name="searchString"
+              placeholder="Search for comics"
+            />
+            <button
+              className="bg-blue-munsell text-white px-4 rounded-lg"
+              type="submit"
+            >
+              Go
+            </button>
+            {errors.searchString && touched.searchString ? (
+              <p className="text-white text-sm mt-2 pl-2">
+                {errors.searchString}
+              </p>
+            ) : null}
+          </Form>
+        )}
+      </Formik>
+      {searchedComicName && (
+        <SearchComicsByNameComponent comicName={searchedComicName} />
+      )}
+      <h2 className="text-white text-lg font-semibold">Quick filters</h2>
+      <h3 className="text-white text-md mt-3 mb-3">Genres</h3>
+      <div className="genres-container flex flex-row gap-3 flex-wrap mb-12">
+        {comicGenres.map((comicGenre, index) => (
+          <label
+            key={comicGenre.toLowerCase()}
+            className={`text-white text-sm rounded-2xl flex flex-row items-center gap-2 ${colourCodeGenreLabels[index]} px-3 py-2`}
+            onClick={() => handleGenreClick(comicGenre)}
+          >
+            {comicGenreIcons[index]}
+            {comicGenre}
+          </label>
+        ))}
+      </div>
+      {selectedGenre && <SearchComicsByGenreComponent genre={selectedGenre} />}
```

Comment:
Not sure if you need the "Component" suffix on the component name 😅. But if it's to help you recongize it better then that's cool

Reply:
Removed them 😄